### DISCUSSION TITLE
bash: flixel: command not found

### DISCUSSION THREAD

**Discussion by goldenPiGames** [HaxeFlixel/flixel#2457] [Q&A]
Just downloaded and went through all the commands in https://haxeflixel.com/documentation/install-haxeflixel/

But flixel commands don't work. For example, after running `haxelib run flixel-tools setup`:

```
Your current settings:

Default Editor                  : Visual Studio Code
Auto open with IDE              : true

flixel-tools setup completed.
Try the 'flixel' command to test it! :)

xxxx@LAPTOP-XXXXXXXX MINGW64 /c/Haxe
$ flixel
bash: flixel: command not found
```
I feel this is some kind of silly issue with my computer configurations somehow, but can someone please help me out?

**Comment by Geokureli** [comment]
Personally I don't use flixel-tools or lime-tools I just always do `haxelib run flixel`. what happens when you run that?
